PowerToys.sln

```diff
@@ -228,6 +269,7 @@ Global
 		{51D3BD1F-07A8-48EB-B2A0-0A249CD4E1A6}.Debug|x64.Build.0 = Debug|x64
 		{51D3BD1F-07A8-48EB-B2A0-0A249CD4E1A6}.Release|x64.ActiveCfg = Release|x64
 		{51D3BD1F-07A8-48EB-B2A0-0A249CD4E1A6}.Release|x64.Build.0 = Release|x64
+<<<<<<< HEAD

```

Comment:
@arjunbalgovind :  Looks like a bad merge?

Reply:
Surprisingly the solution was still loading and building fine even with this.
Fixed. Also verified that it builds debug/release, runs and unit tests pass.